Alignment/OfflineValidation/python/TkAlAllInOneTool/betterConfigParser.py

```diff
@@ -153,6 +153,8 @@ def getGeneral( self ):
         if not self.has_option(internal_section, "workdir"):
             self.set(internal_section, "workdir", "/tmp/$USER")
         general["workdir"] = self.get(internal_section, "workdir")
+        for dir in "workdir", "datadir", "logdir", "eosdir":
+            general[dir] = os.path.expandvars(general[dir])
         return general
```

Comment:
I can fix it in the next iteration, I think at this point it's not worth holding.  It's right before the return, so there's no danger of trying to use dir().